src/process_utility.c

```diff
@@ -1712,16 +1722,32 @@ process_utility_AtEOXact_abort(XactEvent event, void *arg)
 	}
 }
 
+static void
+process_utility_AtSubXact_abort(SubXactEvent event, SubTransactionId mySubid,
```

Comment:
We should stick to lower-case-underscore naming for functions and fix the other callback.

Reply:
fixed